extmap.go

```diff
@@ -0,0 +1,88 @@
+package sdp
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/pions/webrtc/pkg/rtcerr"
+
+	"github.com/pkg/errors"
+)
+
+//ExtMap represents the activation of a single RTP header extension
+type ExtMap struct {
+	Value     int
+	Direction Direction
+	URI       *url.URL
+	ExtAttr   *string
```

Comment:
not sure, i'll think about this

Reply:
@maxhawkins This stuff is coming from my style. I personally prefer the pointer style for the optional stuff. So I will guess that is why I use that here but I really don't remember if there might be an additional special reason. I wrote this months ago. So lets assume that it is because of my style.